packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -42,39 +59,34 @@
   @apply ps-0;
 }
 
-:host(:active) .anchor {
-  @apply text-color-1;
-}
-
 :host([active]) .anchor {
-  @apply text-color-1;
-  border-color: var(--calcite-color-brand);
-  --calcite-icon-color: var(--calcite-color-brand);
+  --calcite-icon-color: var(--calcite-navigation-logo-icon-color-press, var(--calcite-color-brand));
+  border-block-end-color: var(--calcite-navigation-logo-border-color, var(--calcite-color-brand));
 }
 
 .container {
   @apply flex
   flex-col
   truncate
-  text-start
-  px-4
-  mt-0.5;
+  text-start;
+  padding-inline: var(--calcite-spacing-xl);
+  margin-block-start: var(--calcite-spacing-base);
 }
 
 .heading {
-  @apply text-0
-  ms-0
-  truncate
-  text-color-1
-  font-medium;
+  @apply ms-0 truncate;
+  font-weight: var(--calcite-font-weight-medium);
+  color: var(--calcite-navigation-logo-heading-text-color, var(--calcite-color-text-1));
+  font-size: var(--calcite-font-size-0);
```

Comment:
remove these changes except for 
`color: var(--calcite-navigation-logo-heading-text-color, var(--calcite-color-text-1));`